src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -57,8 +57,9 @@ def upload_file(
         filepath: Path,
     ) -> UploadFailure | None:
         """Uploads a file to the ingest server."""
-        filename = urllib.parse.quote(str(filepath))
-        url = f"{self.dsp_ingest_url}/projects/{self.shortcode}/bulk-ingest/ingest/{filename}"
+        url = f"{self.dsp_ingest_url}/projects/{self.shortcode}/bulk-ingest/ingest/{urllib.parse.quote(str(filepath))}"
+        headers = {"Content-Type": "application/octet-stream"}
+        timeout = 60
```

Comment:
If everyone agrees with this approach, we should implement the same for `DspIngestClientLive`, see https://linear.app/dasch/issue/DEV-3761/add-logging-of-requests-made-in-dspingestclientlive